crates/astria-composer/src/executor/bundle_factory/tests.rs

```diff
@@ -230,10 +238,55 @@ mod bundle_factory_tests {
         assert_eq!(actual_seq_action.data, seq_action0.data);
     }
 
+    #[test]
+    fn try_push_full_sanity_check() {
+        // create a bundle factory with max bundle size as 100 bytes
+        let mut bundle_factory = BundleFactory::new(100, 1);
+
+        // push a sequence action that is 100 bytes total
+        let seq_action0 = SequenceAction {
+            rollup_id: RollupId::new([0; ROLLUP_ID_LEN]),
+            data: vec![0; 100 - ROLLUP_ID_LEN],
+            fee_asset_id: default_native_asset_id(),
+        };
+        bundle_factory.try_push(seq_action0.clone()).unwrap();
+
+        // push another sequence action that is <100 bytes total to force the current bundle to
+        // flush
+        let seq_action1 = SequenceAction {
+            rollup_id: RollupId::new([1; ROLLUP_ID_LEN]),
+            data: vec![1; 100 - ROLLUP_ID_LEN],
+            fee_asset_id: default_native_asset_id(),
+        };
+        bundle_factory.try_push(seq_action1).unwrap();
```

Comment:
can we assert that the test results to an `Ok(())`. otherwise any other changes in code might result in a panic

Reply:
thats what `unwrap` does here.

Reply:
yeah but i think `unwrap` panics right? an assertion feels much more nicer if the test fails again?